device_backends/LogicalNameMapping/src/LogicalNameMappingBackend.cc

```diff
@@ -233,6 +236,8 @@ namespace ChimeraTK {
         lnmInfo.nChannels = target_info.getNumberOfChannels();
       }
       if(lnmInfo.length == 0) lnmInfo.length = target_info.getNumberOfElements();
+
```

Comment:
Yes, this part was actually missing. Only the copy was being modified. Seem the tests did not catch that this was missing.